content/ADR-288-deep-link-token-authentication.md

```diff
@@ -0,0 +1,433 @@
+8. The AUTH CHAIN enables cryptographic signing capabilities within Decentraland
+
+```mermaid
+sequenceDiagram
+    participant User as 👤 User
+    participant Client as 💻 Decentraland Client
+    participant Browser as 🌐 Browser
+    participant Server as 🔒 Auth Server
+
+    User->>Client: Presses "Sign In" button
+    Client->>Client: Generates session & displays verification code (e.g., "67")
+    Client->>Browser: Opens auth URL with session token
+    User->>Browser: Completes social login (Google, etc.)
+    Browser->>User: Shows "Verify code: is it 67? YES/NO"
+    User->>User: Manually checks code displayed in Client
+    User->>Browser: Clicks YES
+    Browser->>Server: Confirms verification
+    Server->>Client: Sends AUTH CHAIN
+    Client->>Client: User authenticated
+```
+
+### User Experience Issues
+
+The current verification code flow has significant user experience friction:
+
+- **Multiple context switches**: Users must switch between the Decentraland client and web browser multiple times
+- **Manual verification step**: Users must manually read and confirm the verification code, adding cognitive load
+- **Confusing for new users**: The verification process is not intuitive, especially for first-time users
+- **High entry flow drop rate**: The complex multi-step process results in a high drop-off rate during onboarding
+
+This friction contributes to a **high entry flow drop rate**, where users abandon the sign-in process before completion, negatively impacting user acquisition and retention. Reducing this friction is critical for improving the onboarding experience and reducing user abandonment.
+
+### Security Vulnerability: Session Hijacking via URL Sharing
+
+The current implementation is vulnerable to **URL-based session fixation/hijacking attacks**. A malicious user can execute this attack:
+
+1. Malicious user presses "Sign In" on their Decentraland client
+2. Malicious user's client generates a session and displays a verification code
+3. Malicious user sends the authentication URL to a legitimate user (social engineering)
+4. Legitimate user opens the URL and completes their social login
+5. Legitimate user sees "Verify code: is it [code]? YES/NO" and clicks YES
+6. Server sends the AUTH CHAIN to the **malicious user's client**
+7. Malicious user's client is now authenticated with the **legitimate user's identity**
+8. Malicious user can now impersonate the legitimate user with full signing privileges
+
+This vulnerability exists because:
+- The authentication state is tied to a URL that can be transferred
+- Anyone who completes the social login gets their identity bound to the client that generated the URL
+- No cryptographic binding between the web session and the specific client/user who should authenticate
+- The server cannot verify that the person completing authentication is the same as the person who should own that client session
+
+#### Attack Scenario Diagram
+
+```mermaid
+sequenceDiagram
```

Comment:
Sames as before, as we're already describing the process in the bullet points, what do you think about including only one of them?